test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -935,6 +935,42 @@ def forward(self, x, y):
 
         torch.set_default_dtype(prev_default)
 
+    # In scripting x, y do not carry shape and dtype info.
+    # The following test only works when onnx shape inference is enabled.
+    @skipIfONNXShapeInference(False)
+    def test_true_div_script(self):
```

Comment:
Looks good, thanks.
Should we enable scripting using the decorator?

Reply:
actually this cannot use the decorator. If so the tracing test would also be skipped if onnx shape inference is not turned on.